### DISCUSSION TITLE
Problem connecting to OpenEphys acquisition board

### DISCUSSION THREAD

**Discussion by theparkerlab** [bonsai-rx/bonsai#1675] [Q&A] [answered]
Hello,

I'm using the standard OpenEphys example from the Bonsai Gallery and I'm unable to connect to my OpenEphys acquisition board. I have a newer (OpenEphys FPGA) board that is working fine with the OpenEphys GUI. I have tried the rhd2000_usb3.bit file in both the Bonsai example folder and in the OpenEphys folder and I still get the error message "Runtime Error: Device could not be opened. Is one connected?" Is there a different bit file I should be using now? Or does anyone have ideas as to why I cannot get Bonsai to recognize the board?

Thanks!
Phil

**Comment by theparkerlab** [comment]
Hello,

I found the [instructions](https://open-ephys.github.io/acq-board-docs/User-Manual/New-FPGA.html#newfpga-driver) on the open ephys website for the new module, and found the new "community" package for the OE FPGA acquisition board. When I try to start the workflow, nothing seems to be happening and Bonsai is not throwing any errors. So it seems it can communicate with the board but is not acquiring any data as nothing is coming up on the visualizers for the AcquisitionBoard or AmplifierData nodes. Also, nothing seems to be printed in the command prompt (please see attached pic).

Phil
![bonsai-oe](https://github.com/bonsai-rx/bonsai/assets/139789779/7dc50a32-8fd9-4f12-9535-fff0d0a56a99)

**Reply by bruno-f-cruz** [reply]
The package you are using is not part of this organization. You might have better luck asking on the respective repo.
https://github.com/open-ephys/Bonsai.ONIX/issues

**Reply by theparkerlab** [reply]
Thanks Bruno, I've opened an issue on the [Open Ephys Bonsai Repository](https://github.com/open-ephys/Bonsai.OpenEphys/issues/2).

Phil

**Reply by theparkerlab** [reply, marked as answer]
OEPS helped me solve this problem by downgrading the current board driver version.

"The USB drivers were recently updated by the manufacturer so probably the Bonsai package needs to be repackaged with those...I'm now thinking that if this is the issue in the meantime you might be able to get it to work by downgrading to the previous driver version. Here are the instructions:
 
Uninstall version 1.3.0.8 of the driver and install version [1.3.0.4](https://www.ftdichip.com/Driver/D3XX/FTD3XXDriver_WHQLCertified_v1.3.0.4_Installer.exe.zip) (this is a direct link that should start the download for you)."
